src/packer.pkr.hcl

```diff
@@ -46,89 +46,101 @@ variable "skip_create_ami" {
   filters = {
-    name                = "debian-11-amd64-*"
+    name                = "Windows_Server-2022-English-Full-Base-*"
     root-device-type    = "ebs"
     virtualization-type = "hvm"
   }
   most_recent = true
-  owners      = ["136693071363"]
+  owners      = ["amazon"]
   region      = var.build_region
 }
 
 locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 
-source "amazon-ebs" "example" {
+source "amazon-ebs" "windows" {
   ami_block_device_mappings {
     delete_on_termination = true
     device_name           = "/dev/xvda"
     encrypted             = true
     volume_size           = 8
     volume_type           = "gp3"
   }
-  ami_name                    = "example-hvm-${local.timestamp}-x86_64-ebs"
+
+  ami_name                    = "windows-server-2022-${local.timestamp}-x86_64-ebs"
   ami_regions                 = var.ami_regions
   associate_public_ip_address = true
-  encrypt_boot                = true
-  instance_type               = "t3.small"
-  kms_key_id                  = var.build_region_kms
+
+  communicator  = "winrm"
+  encrypt_boot  = true
+  instance_type = "t3.large"
+
+  kms_key_id = var.build_region_kms
+
   launch_block_device_mappings {
     delete_on_termination = true
     device_name           = "/dev/xvda"
     encrypted             = true
     volume_size           = 8
     volume_type           = "gp3"
   }
+
   region             = var.build_region
   region_kms_key_ids = var.region_kms_keys
-  skip_create_ami    = var.skip_create_ami
-  source_ami         = data.amazon-ami.debian_bullseye.id
-  ssh_username       = "admin"
+
+  skip_create_ami = var.skip_create_ami
+  source_ami      = data.amazon-ami.windows_server_2022.id
+
   subnet_filter {
     filters = {
       "tag:Name" = "AMI Build"
     }
   }
+
   tags = {
-    Application        = "Example"
-    Base_AMI_Name      = data.amazon-ami.debian_bullseye.name
+    Application        = "Windows Server 2022"
+    Base_AMI_Name      = data.amazon-ami.windows_server_2022.name
     GitHub_Release_URL = var.release_url
-    OS_Version         = "Debian Bullseye"
+    OS_Version         = "Windows Server 2022"
     Pre_Release        = var.is_prerelease
     Release            = var.release_tag
     Team               = "VM Fusion - Development"
   }
+
   # Many Linux distributions are now disallowing the use of RSA keys,
   # so it makes sense to use an ED25519 key instead.
   temporary_key_pair_type = "ed25519"
+
+  user_data_file = "src/winrm_bootstrap.txt"
+
   vpc_filter {
     filters = {
       "tag:Name" = "AMI Build"
     }
   }
-}
 
-build {
-  sources = ["source.amazon-ebs.example"]
+  winrm_insecure = true
+  winrm_timeout  = "20m"
+  winrm_use_ssl  = true
+  winrm_username = "Administrator"
 
-  provisioner "ansible" {
-    playbook_file = "src/upgrade.yml"
-  }
+  # Use password set by WinRM. This is temporary, will be removed before release.
+  winrm_password = "SuperS3cr3t!!!!"
```

Comment:
@mcdonnnj i can create a gh actions step to process the user-file with the password and pull it from a temporary github secret. I can update that script later to pull the password from SSM and remove the github secret when its no longer used.

Otherwise, I can to wait until the ssm password is set up and i can just go ahead and pull the password from there as another commit to this pull request. Although I was hoping to put that work in a separate pull request to keep defined tasks seperate.

Let me know your thoughts!

Reply:
Processing the user-data file would work. Would you make an issue around that? I don't want to hold up this pull request on that, but I do want to document it for future work.

Reply:
@mcdonnnj Sounds good. I created an issue here: https://github.com/cisagov/windows-commando-packer/issues/17

Reply:
@itsmostafa Perfect, thanks!